sentences/nl/homeassistant_HassTurnOn.yaml

```diff
@@ -4,6 +4,7 @@ intents:
     data:
       - sentences:
           - <doe> <name> [<naar>] aan
```

Comment:
you can make `<doe>` optional instead of adding another sentence.
So remove the added sentences and change `<doe>` in the existing sentence to `[<doe>]`